C/c4Database.cc

```diff
@@ -73,7 +73,7 @@ static void ensureConfigDirExists(const C4DatabaseConfig2& config) {
 
 static C4DatabaseConfig newToOldConfig(const C4DatabaseConfig2& config2) {
     return C4DatabaseConfig{config2.flags | kC4DB_AutoCompact, NULL,
-                            (config2.flags & kC4DB_VersionVectors) ? kC4VectorVersioning : kC4TreeVersioning_v2,
+                            (config2.flags & kC4DB_VersionVectors) ? kC4VectorVersioning : kC4TreeVersioning,
```

Comment:
This is really strange -- I did not think we were using kC4TreeVersioning_v2 at all anymore. Looks like I did this by accident back in commit ea82fb755 in 2021.